assets/src/design-system/components/toggle/stories/index.js

```diff
@@ -78,55 +79,13 @@ export const _default = () => {
   };
 
   return (
-    <>
-      <Headline as="h1">{'Toggle'}</Headline>
-      <br />
-      <Container>
-        <Row>
-          <Text>{'Normal'}</Text>
-          <Text>{'Normal'}</Text>
-          <Text>{'Disabled'}</Text>
-          <Text>{'Disabled'}</Text>
-        </Row>
-        <Row>
-          <Toggle
-            id="one-light"
-            aria-label="toggle-one-light"
-            name="oneLight"
-            checked={inputState.oneLight}
-            onChange={handleChange}
-            label={text('Toggle 1 Label')}
-          />
-          <Toggle
-            id="two-light"
-            aria-label="toggle-two-light"
-            name="twoLight"
-            checked={inputState.twoLight}
-            onChange={handleChange}
-            label={text('Toggle 2 Label', 'Error')}
-          />
-          <Toggle
-            id="three-light"
-            aria-label="toggle-three-light"
-            name="threeLight"
-            checked={inputState.threeLight}
-            onChange={handleChange}
-            label={text('Toggle 3 Label', 'Disabled')}
-            disabled
-          />
-          <Toggle
-            id="four-light"
-            aria-label="toggle-four-light"
-            name="fourLight"
-            checked={inputState.fourLight}
-            onChange={handleChange}
-            label={text('Toggle 4 Label', 'Disabled')}
-            disabled
-          />
-        </Row>
-      </Container>
-      <DarkThemeProvider>
-        <Container darkMode>
+    <StyleSheetManager
+      stylisPlugins={boolean('isRTL', false) ? [stylisRTLPlugin] : []}
+    >
```

Comment:
diff is weird but this is all I added here

Reply:
See also #6544. Basically, we should rather wrap everything in `<StyleSheetManager>` globally. (not in this PR)